src/material-examples/tabs-template-label/tabs-template-label-example.html

```diff
@@ -1,48 +1,119 @@
-<mat-tab-group class="demo-tab-group">
-  <mat-tab label="Tab 1">
-    <div class="demo-tab-content">
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla venenatis ante augue.
-      Phasellus volutpat neque ac dui mattis vulputate. Etiam consequat aliquam cursus.
-      In sodales pretium ultrices. Maecenas lectus est, sollicitudin consectetur felis nec,
-      feugiat ultricies mi. Aliquam erat volutpat. Nam placerat, tortor in ultrices porttitor,
-      orci enim rutrum enim, vel tempor sapien arcu a tellus.
-    </div>
-  </mat-tab>
-  <mat-tab label="Tab 2">
-    <ng-template mat-tab-label>
-      ⭐
-    </ng-template>
-    <div class="demo-tab-content">
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla venenatis ante augue.
-      Phasellus volutpat neque ac dui mattis vulputate. Etiam consequat aliquam cursus.
-      In sodales pretium ultrices. Maecenas lectus est, sollicitudin consectetur felis nec,
-      feugiat ultricies mi. Aliquam erat volutpat. Nam placerat, tortor in ultrices porttitor,
-      orci enim rutrum enim, vel tempor sapien arcu a tellus.
-    </div>  
-  </mat-tab>
-  <mat-tab label="Tab 3" disabled>
-    No content  
-  </mat-tab>
-  <mat-tab label="Tab 4">
-   <div class="demo-tab-content">
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla venenatis ante augue.
-      Phasellus volutpat neque ac dui mattis vulputate. Etiam consequat aliquam cursus.
-      In sodales pretium ultrices. Maecenas lectus est, sollicitudin consectetur felis nec,
-      feugiat ultricies mi. Aliquam erat volutpat. Nam placerat, tortor in ultrices porttitor,
-      orci enim rutrum enim, vel tempor sapien arcu a tellus.
-      <br />
-      <br />
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla venenatis ante augue.
-      Phasellus volutpat neque ac dui mattis vulputate. Etiam consequat aliquam cursus.
-      In sodales pretium ultrices. Maecenas lectus est, sollicitudin consectetur felis nec,
-      feugiat ultricies mi. Aliquam erat volutpat. Nam placerat, tortor in ultrices porttitor,
-      orci enim rutrum enim, vel tempor sapien arcu a tellus.
-    </div>  
-  </mat-tab>
-  <mat-tab label="Tab 5">
-    No content  
-  </mat-tab>
-  <mat-tab label="Tab 6">
-    No content  
-  </mat-tab>
-</mat-tab-group>
\ No newline at end of file
+<mat-card>
+  <mat-card-content>
+    <h2 class="example-h2">Tabs with text labels</h2>
```

Comment:
Remove `example-h2` since its not a style defined (and isn't prefixed with `demo-`)